torch/_dynamo/variables/functions.py

```diff
@@ -444,6 +444,12 @@ def _setup_exception(self, tx, exc):
             # exception is raised again.
             tracer.exception_handler(e)
 
+    def _is_inside_try_finally(self, tracer):
+        exn_tab_entry = tracer.current_instruction.exn_tab_entry
+        return (sys.version_info >= (3, 11) and exn_tab_entry) or (
+            sys.version_info < (3, 11) and tracer.block_stack
+        )
```

Comment:
This should be a method on the tracer object instead of here so that it can be reused.

Reply:
We don't need this method anymore.